universal_transfer_operator/docs/getting-started/GETTING_STARTED.md

```diff
@@ -0,0 +1,9 @@
+# Getting Started
+
+The Universal Transfer Operator allows data transfers between any Datasets. It copies all the data from the source Dataset to the destination Dataset. The DAG author only needs to use the Universal Transfer Operator for all transfers.
+
+This ensures a consistent set of Data Providers that can read from and write to Datasets. The Universal Transfer
```

Comment:
@sunank200 I think it would be nice if we can explain all the keywords that are specific to UTO. In this case, we can explain - `Data Providers` and `Datasets` or provide a link to a document that does.

Reply:
Also, links to examples would be nice.

Reply:
Can we add a link to them?